### ISSUE TITLE
Error executing command: msgpack decode error [pos 133]: invalid length of bytes for decoding time

### ISSUE THREAD

**Issue by Gooodzi** [cgrates/cgrates#1223]
We are creating account using API
method - ApierV2.SetAccount
entity - {
tenant,
22222,
false,
true
}
/*
   @JsonProperty("Tenant")
    private String tenant;

    @JsonProperty("Account")
    private String account;

    @JsonProperty("AllowNegative")
    private Boolean allowNegative;

    @JsonProperty("ReloadScheduler")
    private Boolean reloadScheduler;
*/

And trying to read it also using API & cgr-console (both ways fails):
method - ApierV2.GetAccount
entity - {
tenant,
22222
}
/*
 @JsonProperty("Tenant")
    private String tenant;

    @JsonProperty("Account")
    private String account;
*/

All these steps works perfectly on e3086a6e15357a253322ebc812ec03838dd2c1b9
And not working on 2524a8c4967c7267a77553946ad3c023daee0d5e

Cgrates config remains unchanged between versions. After returning to e3086a6e15357a253322ebc812ec03838dd2c1b9 - the problem perishes.

Also tested this one on 0dc4c9f3639317c563ddb65360992e0ffed758ff without any changes in config - requests also fail.

Attaching ngrep samples.
[ngrep_api.txt](https://github.com/cgrates/cgrates/files/2410577/ngrep_api.txt)
[ngrep_cgr_console.txt](https://github.com/cgrates/cgrates/files/2410578/ngrep_cgr_console.txt)

**Comment by danbogos** [comment]
Hi @Gooodzi 

Please also attach ngrep for a set API call.

Thanks,
DanB

**Comment by Gooodzi** [comment]
#
T 2018/09/24 11:47:31.750745 10.46.144.3:60601 -> 10.46.79.100:2012 [A
P]
{"id":0,"method":"ApierV2.SetAccount","params":[{"Tenant":"tenant","A
ccount":"34685555555","AllowNegative":true,"ReloadScheduler":true}]}.

##
T 2018/09/24 11:47:31.752684 10.46.79.100:2012 -> 10.46.144.3:60601 [A
P]
{"id":0,"result":"OK","error":null}

#

It is strange, but creating this one went normally, and reading it also.
But regarding the ones from the earlier versions - can't read not a single one of those.
Also found out why creating didn't work before - we've checked firstly if such account is present via Get request (and this one failed)


So tl;dr:
- Set method works fine, get method works only with those accounts that were successfully created by this verion's set.
- nothing changed in the procedure we use for creating accounts\getting account info.

**Comment by Trial97** [comment]
Hi @Gooodzi ,

We could not replicate the issue by using the commands you have sent, driving us to think that you have had the account before the set command with some additional data inside. Please attach getAccount with old commit.

Thanks,
Trial97

**Comment by Gooodzi** [comment]
Hello @Trial97 ,
Here's sample request from old commit:

```
T 2018/09/25 10:03:36.105224 10.46.144.2:61265 -> 10.198.3.19:2012 [A]
......
#
T 2018/09/25 10:03:36.220690 10.46.144.2:61265 -> 10.198.3.19:2012 [AP]
{"id":1,"method":"ApierV2.GetAccount","params":[{"Tenant":"tenant","Account":"34645555555"}]}.

##
T 2018/09/25 10:03:36.222875 10.198.3.19:2012 -> 10.46.144.2:61265 [AP]
{"id":1,"result":{"ID":"tenant:34645555555","BalanceMap":{"*data":[{"Uuid":"d821f4ac-0fd5-4a89-b66a-772810003585","ID":"*bonos_0min_200mb_plan_data","Value":256000,"Directions":{},"ExpirationDate":"2017-10-01T02:06:53Z","Weight":10,"DestinationIDs":{},"RatingSubject":"","Categories":{"data":true},"SharedGroups":{},"Timings":null,"TimingIDs":{},"Disabled":false,"Factor":null,"Blocker":false}],"*monetary":[{"Uuid":"17a28cc0-0dc5-490e-a5c8-f7ba7dfc92e2","ID":"*default","Value":-22.69266,"Directions":null,"ExpirationDate":"0001-01-01T00:00:00Z","Weight":0,"DestinationIDs":null,"RatingSubject":"","Categories":null,"SharedGroups":null,"Timings":null,"TimingIDs":null,"Disabled":false,"Factor":null,"Blocker":false}]},"UnitCounters":null,"ActionTriggers":null,"AllowNegative":true,"Disabled":false},"error":null}

########
T 2018/09/25 10:03:36.391095 10.46.144.2:61265 -> 10.198.3.19:2012 [AF]
......
##
T 2018/09/25 10:03:36.456345 10.46.144.2:61265 -> 10.198.3.19:2012 [A]
```
......
####

Tested also in entirely new environment, and account created in older commit cannot be queried in new commit (versions in 1st post) - although the procedure we use and the CGRates config remains unchanged.

**Comment by Gooodzi** [comment]
Hi again,

A little update - after testing different versions I found a commit that comes first with problem described:
- commit 648f069f8d570cd81527e7eb6df7ce01b1decc3c
After this one - all the commits tested have such a problem
Before this one - everything works as expected, the latest commit without problems is commit 5a9760ea311f163d7f3dc6c99db1a779088d8178

I hope this info will help you in localizing problem

**Comment by Trial97** [comment]
Hello @Gooodzi ,

We have found a problem with the library that encodes "msgpack". Please update from master and retest. 
Also please don't forget to run \: `glide install` before rebuild.

Thanks,
Trial97

**Comment by Gooodzi** [comment]
Hello @Trial97 ,

Thanks, updated to latest commit on test environment, the problems seems to be gone.
I'll monitor and test it yet, but I think the issue is solved.

Thank you again)
